earth_enterprise/src/fusion/autoingest/tools/genewmapdatabase.cpp

```diff
@@ -33,7 +34,7 @@ usage(const std::string &progn, const char *msg = 0, ...)
   }
 
   fprintf(stderr,
-          "\nusage: %s [options] [--meta <key>=<value>]... -o <dbname> [--imagery <imagery project>] [--map <map project>] [--mercator : use mercator projection] [--flat : use flat(Plate Carre) projection (default)]\n"
+          "\nusage: %s [options] [--meta <key>=<value>]... -o <dbname> [--imagery <imagery project>] [--map <map project>] [--mercator : use mercator projection database] [--flat : use flat(Plate Carre) projection database (default)]\n"
```

Comment:
I added it because previously it implied that both database and the imagery were either flat or mercator. With the changes I made the flags really only indicate that the database is flat or mercator. I can revert it if you believe it makes more sense.

Reply:
I think it would be a confusion if we have there. The documentation explains what the command does, so i don't think we need to add there.

Reply:
Reverted back to the original text.